css/properties.json

```diff
@@ -6121,6 +6121,21 @@
     "order": "oneOrTwoValuesLengthAbsoluteKeywordsPercentages",
     "status": "standard"
   },
+  "place-content": {
+    "syntax": "<'align-content'> <'justify-content'>?",
```

Comment:
Just note that this has the gotcha that a single `<align-content>` value that wouldn't be valid as `<justify-content>` is invalid.